### ISSUE TITLE
Highlighted command in quick open command palette doesn't meet 4.5 contrast ratio

### ISSUE THREAD

**Issue by erjohnms** [Microsoft/monaco-editor#879]
monaco-editor version: 0.10.1
Browser: Any
OS: Windows 10

For the currently selected command, the text color is #6C6C6C while the background color is #DCEBFC. These colors have a contrast ratio of 4.3:1 which doesn't meet the accessibility requirement of 4.5:1

![image](https://user-images.githubusercontent.com/19472252/40065978-0b0ae97e-5831-11e8-9c64-2b8d25cfbae1.png)

**Comment by bpasero** [comment]
@alexandrudima is there a place I can add dynamic CSS only to the standalone editor? I think I would need a similar place like `shell.ts` but for the standalone editor to define this rule there. The reason this color is not defined from the tree is to avoid layer breakers (tree knows nothing about highlighted labels).

**Comment by alexandrudima** [comment]
@bpasero Here is the CSS file where I keep all the standalone editor specific things --- https://github.com/Microsoft/vscode/blob/master/src/vs/editor/standalone/browser/standalone-tokens.css

If this is quickOpen specific you could simply add a new CSS file at https://github.com/Microsoft/vscode/blob/master/src/vs/editor/standalone/browser/quickOpen/editorQuickOpen.ts

**Comment by bpasero** [comment]
Fixed via https://github.com/Microsoft/vscode/commit/c958e894992ce19253e2c6fd276f0f7887592bd6
